stable/kommander/values.yaml

```diff
@@ -0,0 +1,5 @@
+kommander:
+  replicas: 1
+  image:
```

Comment:
I would suggest image would be a first level field

Reply:
ack